ext/jsonnet/vm.c

```diff
@@ -272,22 +396,32 @@ rubyjsonnet_init_vm(VALUE mJsonnet)
     rb_define_method(cVM, "gc_growth_trigger=", vm_set_gc_growth_trigger, 1);
     rb_define_method(cVM, "string_output=", vm_set_string_output, 1);
     rb_define_method(cVM, "max_trace=", vm_set_max_trace, 1);
+    rb_define_method(cVM, "fmt_indent=", vm_set_fmt_indent, 1);
+    rb_define_method(cVM, "fmt_max_blank_lines=", vm_set_fmt_max_blank_lines, 1);
+    rb_define_method(cVM, "fmt_string=", vm_set_fmt_string, 1);
+    rb_define_method(cVM, "fmt_comment=", vm_set_fmt_comment, 1);
+    rb_define_method(cVM, "fmt_pad_arrays=", vm_set_fmt_pad_arrays, 1);
+    rb_define_method(cVM, "fmt_pad_objects=", vm_set_fmt_pad_objects, 1);
+    rb_define_method(cVM, "fmt_pretty_field_names=", vm_set_fmt_pretty_field_names, 1);
+    rb_define_method(cVM, "fmt_sort_imports=", vm_set_fmt_sort_imports, 1);
+    rb_define_method(cVM, "fmt_file", vm_fmt_file, 2);
+    rb_define_method(cVM, "fmt_snippet", vm_fmt_snippet, 2);
```

Comment:
Sorry. I overlooked this point at the first comment.

As long as you intend to call these methods from their wrappers, these methods should be private.
Should use `rb_define_private_method` as `eval_file` or `eval_snippet` does.